Missing import in NotificationExampleService.java

Hello Jorge,

if I run:

```
mvn archetype:generate \
  -DarchetypeGroupId=io.github.jorgerojasdev \
  -DarchetypeArtifactId=hexagonal-reactive-archetype \
  -DarchetypeVersion=0.0.1 \
  -DgroupId=com \
  -DartifactId=dcremonini
```
and then

```
mvn install
```

I get a compilation error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project usecase: Compilation failure: Compilation failure: 
[ERROR] /tmp/test1/dcremonini/usecase/src/main/java/com/usecase/notification/service/NotificationExampleService.java:[4,55] package com.libraries.flux.notifier.service.abstraction does not exist
[ERROR] /tmp/test1/dcremonini/usecase/src/main/java/com/usecase/notification/service/NotificationExampleService.java:[6,53] cannot find symbol
```

--- a/src/main/resources/archetype-resources/usecase/src/main/java/__artifactId__/notification/service/NotificationExampleService.java
+++ b/src/main/resources/archetype-resources/usecase/src/main/java/__artifactId__/notification/service/NotificationExampleService.java
@@ -4,7 +4,7 @@
 package ${package}.${artifactId}.notification.service;
 
 import ${package}.domain.example.model.ExampleModel;
-import ${groupId}.libraries.flux.notifier.service.abstraction.Notifier;
+import io.github.jorgerojasdev.libraries.flux.notifier.service.abstraction.Notifier;
 
 public interface NotificationExampleService extends Notifier<ExampleModel> {
 

